services/notifications/src/simcore_service_notifications/services/rabbitmq.py

```diff
@@ -0,0 +1,30 @@
+from collections.abc import AsyncIterator
+from typing import cast
+
+from fastapi import FastAPI
+from fastapi_lifespan_manager import State
+from servicelib.rabbitmq import RabbitMQRPCClient, wait_till_rabbitmq_responsive
+from settings_library.rabbit import RabbitSettings
+
+from ..core.settings import ApplicationSettings
+
+
+async def rabbitmq_lifespan(app: FastAPI) -> AsyncIterator[State]:
+    settings: ApplicationSettings = app.state.settings
+    rabbit_settings: RabbitSettings = settings.NOTIFICATIONS_RABBITMQ
+    app.state.rabbitmq_rpc_server = None
+
+    await wait_till_rabbitmq_responsive(rabbit_settings.dsn)
```

Comment:
i will address next thisone. All this again is possible to put it in servicelib instead of having to copy&paste it again and again